defaults/main.yml

```diff
@@ -264,20 +275,35 @@ clickhouse_invalidcertificatehandler:
 #   realm:
 
 # JDBC Bridge
+clickhouse_jdbc_bridge_handler_on_config_change: "Restart-clickhouse-JDBC-bridge"
```

Comment:
This is a not configurable var, maybe its a better option to set it in a 'vars' folder (same as line 21 one)